docs/UpdaterConfig.md

```diff
@@ -223,17 +223,27 @@ The information is downloaded in a single HTTP request and polled regularly.
 <!-- vehicle-positions BEGIN -->
 <!-- NOTE! This section is auto-generated. Do not change, change doc in code instead. -->
 
-| Config Parameter           |       Type      | Summary                                                                    |  Req./Opt. | Default Value | Since |
-|----------------------------|:---------------:|----------------------------------------------------------------------------|:----------:|---------------|:-----:|
-| type = "vehicle-positions" |      `enum`     | The type of the updater.                                                   | *Required* |               |  1.5  |
-| feedId                     |     `string`    | Feed ID to which the update should be applied.                             | *Required* |               |  2.2  |
-| frequency                  |    `duration`   | How often the positions should be updated.                                 | *Optional* | `"PT1M"`      |  2.2  |
-| url                        |      `uri`      | The URL of GTFS-RT protobuf HTTP resource to download the positions from.  | *Required* |               |  2.2  |
-| [headers](#u__6__headers)  | `map of string` | HTTP headers to add to the request. Any header key, value can be inserted. | *Optional* |               |  2.3  |
+| Config Parameter            |       Type      | Summary                                                                    |  Req./Opt. | Default Value | Since |
+|-----------------------------|:---------------:|----------------------------------------------------------------------------|:----------:|---------------|:-----:|
+| type = "vehicle-positions"  |      `enum`     | The type of the updater.                                                   | *Required* |               |  1.5  |
+| feedId                      |     `string`    | Feed ID to which the update should be applied.                             | *Required* |               |  2.2  |
+| frequency                   |    `duration`   | How often the positions should be updated.                                 | *Optional* | `"PT1M"`      |  2.2  |
+| fuzzyTripMatching           |    `boolean`    | Whether to match trips fuzzily.                                            | *Optional* | `false`       |  2.5  |
+| url                         |      `uri`      | The URL of GTFS-RT protobuf HTTP resource to download the positions from.  | *Required* |               |  2.2  |
+| [features](#u__6__features) |    `enum set`   | Which features of GTFS RT vehicle positions should be loaded into OTP.     | *Optional* |               |  2.5  |
```

Comment:
I think we should make it possible to define default values for this. As far as I know, it's not yet supported for enum sets but I can do it within the scope of this pr if we think it's a good idea.

Reply:
That's a good idea.

Reply:
There were some issues with getting the default values displayed. I dicussed this with @t2gran and we decided to fix that issue in a followup pr.

Reply:
But the configuration now uses default values if custom values are not defined.